cmd/clusterawsadm/cmd/gc/disable.go

```diff
@@ -0,0 +1,83 @@
+/*
+Copyright 2022 The Kubernetes Authors.
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+    http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
+
+package gc
+
+import (
+	"fmt"
+	"path/filepath"
+
+	"github.com/spf13/cobra"
+	"k8s.io/client-go/util/homedir"
+
+	gcproc "sigs.k8s.io/cluster-api-provider-aws/cmd/clusterawsadm/gc"
+	"sigs.k8s.io/cluster-api/cmd/clusterctl/cmd"
+)
+
+func newDisableCmd() *cobra.Command {
+	clusterName := ""
+	namespace := ""
+	kubeConfig := ""
+	kubeConfigDefault := ""
```

Comment:
```suggestion
	var ( 
	    clusterName string
	    namespace string
 	    kubeConfig string
	    kubeConfigDefault string
	)
```

Just think that this is neater. :) But up to you. :)

Reply:
Yes it is, changed.